docs/content/gameobject/world-commands.md

```diff
@@ -32,145 +32,78 @@ The commands documentation is:
-
-Returns: the request Id for this command as a `long`.
-
-The corresponding response callback is `WorldCommandResponseHandler.OnCreateEntityResponse`:
+  * `WorldCommands.CreateEntity.Request request`: The command request payload.
+  * `Action<WorldCommands.CreateEntity.ReceivedResponse> callback`: Optional. A callback that will be called when the command response is received.
 
-```csharp
-event Action<CreateEntity.ReceivedResponse> OnCreateEntityResponse
-```
-
-Callback parameters:
-
-  * `ReceivedResponse`: The data associated with the incoming response stored inside a [`ReceivedResponse`]({{urlRoot}}/content/world-component-commands-requests-responses#receivedresponse) struct.
-
-The `CreateEntity.ReceivedResponse` struct contains the same fields as any `ReceivedResponse` struct. However, in this case, the `EntityId` field represents the entity ID of the newly-created entity. If the command failed, this field is null. You can additionally use the `ReserveEntityIds` world command before entity creation which makes it easier to create multiple entities in a group.
+[//]: # (TODO - explain what “handling the response based on the information contained in this object” means)
 
 ### ReserveEntityIds
 
 Like other commands, a `CreateEntity` command response can time out, but the entity might still be created. As we do not know which ID was used by the SpatialOS Runtime to create this entity, the worker needs to retry the `CreateEntity` command request. This might lead to creating multiple entities as a new entity ID will be used by the SpatialOS Runtime, if the previous `CreateEntity` command did succeed.
 To avoid this, the worker can reserve an entity ID before sending the `CreateEntity` command request. Depending on the command response, you should do the following:
-The command succeeded: The entity got successfully created
-The command failed with the status code `ApplicationError` and the following message: “'Entity reservation failed. The entity with Id <{id}> could not be found. The reservation might have expired or never existed”. If you have used the entity ID that you received from the `ReserveEntityIds` command response for sending the `CreateEntity` commands, then your entity has already been created and you don’t need to retry it anymore.
-The command timed out or another error appeared: retry the command.
-[//]: # (TODO - link to status codes for error messages.)
 
+  * The command succeeded: The entity got successfully created
+  * The command failed with the status code `ApplicationError` and the following message: “'Entity reservation failed. The entity with Id <{id}> could not be found. The reservation might have expired or never existed”. If you have used the entity ID that you received from the `ReserveEntityIds` command response for sending the `CreateEntity` commands, then your entity has already been created and you don’t need to retry it anymore.
```

Comment:
```suggestion
  * The command failed with the status code `ApplicationError` and the following message: `Entity reservation failed. The entity with Id <{id}> could not be found. The reservation might have expired or never existed`. 
  <br/>If you have used the entity ID that you received from the `ReserveEntityIds` command response for sending the `CreateEntity` commands, then your entity has already been created and you don’t need to retry it anymore.
```
I know you hate BRs but it helps break up the message and the action.

Reply:
entity Id instead of entity ID?